lib/tasks/full_import.rake

```diff
@@ -0,0 +1,405 @@
+    begin
+      po = User.find_or_initialize_by(email: @row[:project_owner_email].strip.downcase)
+      name_tokens = @row[:project_owners_name].split(" ")
+      po.first_name = name_tokens[0] || "ERROR"
+      po.last_name = name_tokens[1] || "ERROR"
+      po.phone = @row[:project_owner_phone] || "ERROR"
+      po.role = "user"
+      po.heard_about_us = "IMPORT"
+      po.password = DEFAULT_PASSWORD
+      po.password_confirmation = DEFAULT_PASSWORD
+      po.skip_confirmation!
+      po.save!
+      log("CREATED Project Owner user #{po.id} #{@row[:project_owners_name]}")
+    rescue StandardError => e
+      log("ERROR #{e} #{@row[:project_owner_email]}")
+      return nil
+    end
+
+    # Finisher
+    begin
+      finisher = User.find_or_initialize_by(email:  @row[:finisher_email].strip.downcase)
+      name_tokens = @row[:finisher_name].split(" ")
+      finisher.first_name = name_tokens[0] || "ERROR"
+      finisher.last_name = name_tokens[1] || "ERROR"
+      finisher.phone = @row[:finisher_phone] || "ERROR     " # must be 10 char
+      finisher.role = "user"
+      finisher.heard_about_us = "IMPORT"
+      finisher.password = DEFAULT_PASSWORD
+      finisher.skip_confirmation!
+      finisher.save!
+      log("CREATED Finisher user #{finisher.id} #{@row[:finisher_name]}")
+    rescue StandardError => e
+      log("ERROR #{e} #{@row[:finisher_email]}")
+      return nil
+    end
+
+    return [po, finisher]
+  end
+
+  def create_project!(po)
+    project = Project.find_or_initialize_by(name: "#{@row[:project_name]} [IMPORT]")
+
+    project.assign_attributes(
+      user_id: po.id,
+      status: "IN PROCESS: UNDERWAY",
+      description: @row[:project_description],
+      street: @row[:project_owner_address],
+      street_2: nil,
+      city: @row[:city],
+      state: @row[:state],
+      country: @row[:country] || 'US',
+      postal_code: @row[:zip],
+      craft_type: @row[:craft],
+      has_pattern: @row[:pattern],
+      material_type: @row[:material],
+      crafter_name: @row[:original_crafter],
+      crafter_description: @row[:crafter_bio],
+      recipient_name: @row[:who_for],
+      more_details: nil,
+      can_publicize: nil,
+      terms_of_use: nil,
+      phone_number: @row[:project_owner_phone],
+      in_home_pets: nil,
+      has_smoke_in_home: nil,
+      no_smoke: nil,
+      no_cats: nil,
+      no_dogs: nil,
+      crafter_dominant_hand: nil,
+      manager: MANAGER_JEAN,
+      joann_helped: nil,
+      urgent: nil,
+      influencer: nil,
+      group_project: nil,
+      press: nil,
+      privacy_needed: nil,
+      group_manager_id: nil,
+      press_region: nil,
+      press_outlet: nil,
+      can_use_first_name: nil,
+      can_share_crafter_details: nil,
+      material_brand: @row[:brands],
+      has_materials: nil,
+      needs_attention: nil,
+      dominant_hand: 'unknown'
+    )
+    project.ensure_inbound_email_address
+
+    begin
+      project.save! validate: false
```

Comment:
Will this create Project objects that result in web errors when we try to view them? Or only the case where attempting to edit requires the manger to fill in some fields?